setup.cfg

```diff
@@ -1,3 +1,78 @@
+[metadata]
+name = cobra
+url = https://opencobra.github.io/cobrapy
+download_url = https://pypi.org/project/cobra
+project_urls =
+    Source Code = https://github.com/opencobra/cobrapy
+    Documentation = https://cobrapy.readthedocs.io
+    Bug Tracker = https://github.com/opencobra/cobrapy/issues
+author = The cobrapy core development team.
+author_email = cobra-pie@googlegroups.com
+maintainer = Moritz E. Beber
+maintainer_email = moritz.beber@gmail.com
+# Please consult https://pypi.org/classifiers/ for a full list.
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)
+    License :: OSI Approved :: GNU Lesser General Public License v2 or later (LGPLv2+)
+    Natural Language :: English
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Scientific/Engineering :: Bio-Informatics
+license = LGPL-2.0
+description = COBRApy is a package for constraint-based modeling of metabolic networks.
+long_description = file: README.rst, INSTALL.rst
+long_description_content_type = text/x-rst
+keywords =
+    metabolism
+    biology
+    constraint-based
+    linear programming
+    mixed-integer
+    optimization
+    flux-balance analysis
+    reconstruction
+
+[options]
+zip_safe = True
+install_requires =
+    depinfo
+    future
+    numpy ~= 1.13
```

Comment:
I would prefer minimum versions instead of locked minor versions. We often go a while without releases and that might block security updates for other packages. Also I think we can lift the lock fro python-libsbml-experimental. The recent versions always had wheels.

Reply:
In our current setup we use `numpy >=1.13`. This notation `numpy ~= 1.13` is a shorthand for `numpy >=1.13,<2`. Since major versions are allowed to contain breaking API changes I thought it's a good idea to be cautious here.

I'm less certain about python-libsbml-experimental, in the past, creation of the wheels has often been delayed by days to weeks, leading to issues for users with libSBML compilation. I would see how this goes in the next two releases before feeling confident in removing the pin.

Reply:
Oh yeah my bad, I misunderstood the docs and thought that would enforce numpy 1.13.x. If it only looks major versions that's perfect with me.